packages/lexical/src/LexicalEvents.ts

```diff
@@ -542,7 +542,7 @@ function onPointerDown(event: PointerEvent, editor: LexicalEditor) {
   // TODO implement text drag & drop
   const target = event.target;
   const pointerType = event.pointerType;
-  if (isDOMNode(target) && pointerType !== 'touch' && event.button === 0) {
+  if (isDOMNode(target) && pointerType !== 'touch' && pointerType !== 'pen' && event.button === 0) {
```

Comment:
This is needed to pass the formatting checks

```suggestion
  if (
    isDOMNode(target) &&
    pointerType !== 'touch' &&
    pointerType !== 'pen' &&
    event.button === 0
  ) {
```